README.md

```diff
@@ -1,3 +1,4 @@
 # Hello World!
 Getting started with Hello World!
 First README comment
+Second README comment
```

Comment:
This is a great change. I've been waiting for it since forever! Please merge as soon as possible!